test/reductions/exponentiated_gradient/test_expgrad_smoke.py

```diff
@@ -2,94 +2,88 @@
+        self.X = pd.DataFrame({"X1": X1, "X2": X2, "X3": X3})
+        self.y = pd.Series(labels)
+        self.A = pd.Series(aux_data)
         self.learner = simple_learners.LeastSquaresBinaryClassifierLearner()
         self._PRECISION = 1e-6
 
-    smoke_test_data = [{"cons_class": moments.DP, "eps": 0.100,
+    smoke_test_data = [{"cons_class": moments.DemographicParity, "eps": 0.100,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.100000,
                         "error": 0.250000, "n_oracle_calls": 32,
                         "n_classifiers": 3},
-                       {"cons_class": moments.DP, "eps": 0.050,
+                       {"cons_class": moments.DemographicParity, "eps": 0.050,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.050000,
                         "error": 0.266522, "n_oracle_calls": 23,
                         "n_classifiers": 6},
-                       {"cons_class": moments.DP, "eps": 0.020,
+                       {"cons_class": moments.DemographicParity, "eps": 0.020,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.020000,
                         "error": 0.332261, "n_oracle_calls": 22,
                         "n_classifiers": 5},
-                       {"cons_class": moments.DP, "eps": 0.010,
+                       {"cons_class": moments.DemographicParity, "eps": 0.010,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.010000,
                         "error": 0.354174, "n_oracle_calls": 22,
                         "n_classifiers": 5},
-                       {"cons_class": moments.DP, "eps": 0.005,
+                       {"cons_class": moments.DemographicParity, "eps": 0.005,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.005000,
                         "error": 0.365130, "n_oracle_calls": 22,
                         "n_classifiers": 5},
-                       {"cons_class": moments.EO, "eps": 0.100,
+                       {"cons_class": moments.EqualizedOdds, "eps": 0.100,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.100000,
                         "error": 0.309333, "n_oracle_calls": 21,
                         "n_classifiers": 4},
-                       {"cons_class": moments.EO, "eps": 0.050,
+                       {"cons_class": moments.EqualizedOdds, "eps": 0.050,
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.050000,
                         "error": 0.378827, "n_oracle_calls": 19,
                            "n_classifiers": 6},
-                       {"cons_class": moments.EO, "eps": 0.020,
+                       {"cons_class": moments.EqualizedOdds, "eps": 0.020,
```

Comment:
I thought I'd already made all those changes?

Reply:
Correct, maybe I did something weird while merging?

Reply:
Yes, there are certainly a few warnings at the bottom of this PR. Let me merge and see what happens.